src/actions/authActions.js

```diff
@@ -74,7 +74,7 @@ export const setToken = async (token, shouldSave) => {
             mutation: GET_EMAIL,
             variables: { token },
         });
-        const email = verifyToken.payload.username;
+        let email = usernameEmail || verifyToken.payload.username;
```

Comment:
why is this a `let` ? does this get changed later? we should be using const whenever possible